*** browser/installer/package-manifest.in	2019-01-19 12:02:27.743842399 +0100
--- browser/installer/package-manifest.in.new	2019-01-19 12:03:16.662609341 +0100
***************
*** 590,607 ****
  #endif
  
  ; [Browser Chrome Files]
! @BINPATH@/browser/extensions/tprb.addon@searxes.danwin1210.me/*
! @BINPATH@/browser/extensions/tortm-browser-button@jeremybenthum/*
! @BINPATH@/browser/extensions/SubmitMe@0xbeef.coffee/*
! @BINPATH@/browser/extensions/SimpleSumOfUs@0xbeef.coffee/*
! @BINPATH@/browser/extensions/rsf@0xbeef.coffee/*
! @BINPATH@/browser/extensions/rnrMcDonalds@0xbeef.coffee/*
! @BINPATH@/browser/extensions/LibrejsDrive@0xbeef.coffee/*
! @BINPATH@/browser/extensions/jid1-KtlZuoiikVfFew@jetpack/*
! @BINPATH@/browser/extensions/https-everywhere@eff.org/*
! @BINPATH@/browser/extensions/goteo@0xbeef.coffee/*
! @BINPATH@/browser/extensions/FreeUSPS@0xbeef.coffee/*
! @BINPATH@/browser/extensions/DMCAreg@0xbeef.coffee/*
  @RESPATH@/browser/chrome.manifest
  @RESPATH@/browser/chrome/browser@JAREXT@
  @RESPATH@/browser/chrome/browser.manifest
--- 590,607 ----
  #endif
  
  ; [Browser Chrome Files]
! @RESPATH@/browser/extensions/tprb.addon@searxes.danwin1210.me/*
! @RESPATH@/browser/extensions/tortm-browser-button@jeremybenthum/*
! @RESPATH@/browser/extensions/SubmitMe@0xbeef.coffee/*
! @RESPATH@/browser/extensions/SimpleSumOfUs@0xbeef.coffee/*
! @RESPATH@/browser/extensions/rsf@0xbeef.coffee/*
! @RESPATH@/browser/extensions/rnrMcDonalds@0xbeef.coffee/*
! @RESPATH@/browser/extensions/LibrejsDrive@0xbeef.coffee/*
! @RESPATH@/browser/extensions/jid1-KtlZuoiikVfFew@jetpack/*
! @RESPATH@/browser/extensions/https-everywhere@eff.org/*
! @RESPATH@/browser/extensions/goteo@0xbeef.coffee/*
! @RESPATH@/browser/extensions/FreeUSPS@0xbeef.coffee/*
! @RESPATH@/browser/extensions/DMCAreg@0xbeef.coffee/*
  @RESPATH@/browser/chrome.manifest
  @RESPATH@/browser/chrome/browser@JAREXT@
  @RESPATH@/browser/chrome/browser.manifest
